src/ray/raylet/scheduling_resources.cc

```diff
@@ -135,9 +135,15 @@ double ResourceSet::GetNumCpus() const {
 
 const std::string ResourceSet::ToString() const {
   std::string return_string = "";
```

Comment:
are you trying to generate a json-compatible string here? I wonder if we should just use the json library or add a ToJson() method that explicitly does that.

Reply:
The goal was not to make a JSON compatible string, but rather to make the printing nicer. This changes it from `{GPU,1.000000}, {CPU,1.000000}, ` (note the extra comma and space at the end) to `{GPU,1.000000}, {CPU,1.000000}`.